test/e2e/container_clone_test.go

```diff
@@ -304,4 +304,11 @@ var _ = Describe("Podman container clone", func() {
 		Expect(session.OutputToString()).Should(ContainSubstring("12=3"))
 
 	})
+
+	It("podman container clone container with healthcheck", func() {
+		podmanTest.PodmanExitCleanly("run", "-d", "--rm", "--health-cmd", "true", "--name", "parent", ALPINE, "sleep", "200")
+		podmanTest.PodmanExitCleanly("healthcheck", "run", "parent")
+		podmanTest.PodmanExitCleanly("container", "clone", "--run", "parent", "clone", ALPINE)
+		podmanTest.PodmanExitCleanly("healthcheck", "run", "clone")
```

Comment:
Maybe set a startup hc as well and the hc failure action, then inspect the cloned container to ensure all settings have been copied. Right now we only know the some hc check exists but not that it is the "right" one.